templates/alertmanager.yaml

```diff
@@ -49,6 +49,7 @@ spec:
   secrets:
     - alertmanager-k8s-tls
     - alertmanager-k8s-proxy
+  priorityClassName: "rhoam-pod-priority"
```

Comment:
Should this the same as what's in the `prometheus.yaml` file? E.g:
```yaml
{{- if .PriorityClassName}}
  priorityClassName: {{.PriorityClassName}}
{{- end}}
```

Reply:
Yeah, good catch, Updated now.